### ISSUE TITLE
Freeze .NET version until released? (or at least show what version is it based on!)

### ISSUE THREAD

**Issue by alex-jitbit** [KirillOsenkov/SourceBrowser#243]
I was browsing `source.dot.net` and found a method overload that does not exist in .NET 8.

Apparently the website is using .NET 9 preview?

Can you either freeze the version until it's publicly released OR indicate whichever version the website is based on?

**Comment by KirillOsenkov** [comment]
No, unfortunately we can only index whatever is in the main branch. Sorry.

**Comment by alex-jitbit** [comment]
It'd be great if you could please at least add a note, which sources the site is based upon. Спасибо ❤️
